_includes/_bitmovin-player.html

```diff
@@ -6,12 +6,13 @@
   {% assign is_card = false %}
 {% endif %}
 
-{% assign timestamps = video.timestamps | jsonify %}
+{% assign timestamps = video.timestamps | jsonify | escape %}
 
 {% if recommended_collection.size > 0 %}
   {% assign sorted_collection = recommended_collection | sort: 'date' | reverse %}
-  {% assign json_collection = sorted_collection | jsonify_message: page.date %}
+  {% assign json_collection = sorted_collection | jsonify_message: page.date | escape %}
   <crds-bitmovin-player 
+    recommended-collection='{{json_collection}}' 
```

Comment:
This update seems unrelated. Can you double check that this edit was intentional?

Reply:
@tcmacdonald  So I did all my work, rebased to squash all my commits into a single commit, and then rebased off of master. I didn't notice it before, but for some reason when I pushed to github, two other commits by JerebChase were included in this PR. I don't know why that would be. Any thoughts? What do I need to do to remove his commits from my PR?

Reply:
Got it. This update may not be reflected in `development` and when you pulled down `master` you introduced it. 

As long as this is consistent with what we've got in the `master` branch, I think its safe to leave as-is.